docs/aws/audit/route53monitoring/rules/config_enabled_remediation.mdx

```diff
@@ -1,6 +1,240 @@
+   - Choose an IAM role that AWS Config can use to access your resources. If you don't have an existing role, AWS Config can create one for you.
+
+4. **Enable AWS Config:**
+   - Review your settings and click "Confirm" to enable AWS Config.
+   - AWS Config will now start recording configuration changes for Route 53 and other selected resources.
+
+By following these steps, you ensure that AWS Config is enabled and monitoring Route 53 configurations, helping to prevent misconfigurations.
+</Accordion>
+
+<Accordion title='Using CLI'>
+To ensure AWS Config is enabled for Route 53 using AWS CLI, follow these steps:
+
+1. **Create an S3 Bucket for AWS Config Logs:**
+   AWS Config requires an S3 bucket to store configuration history and snapshots. Create an S3 bucket if you don't already have one.
+   ```sh
+   aws s3api create-bucket --bucket my-config-bucket --region us-east-1
+   ```
+
+2. **Create an IAM Role for AWS Config:**
+   AWS Config needs an IAM role with the necessary permissions to access your resources. Create a role with the required policies.
+   ```sh
+   aws iam create-role --role-name AWSConfigRole --assume-role-policy-document file://trust-policy.json
+   aws iam attach-role-policy --role-name AWSConfigRole --policy-arn arn:aws:iam::aws:policy/service-role/AWSConfigRole
+   ```
+
+   The `trust-policy.json` should look like this:
+   ```json
+   {
+     "Version": "2012-10-17",
+     "Statement": [
+       {
+         "Effect": "Allow",
+         "Principal": {
+           "Service": "config.amazonaws.com"
+         },
+         "Action": "sts:AssumeRole"
+       }
+     ]
+   }
+   ```
+
+3. **Enable AWS Config:**
+   Use the `put-configuration-recorder` command to enable AWS Config and specify the IAM role and S3 bucket.
+   ```sh
+   aws configservice put-configuration-recorder --configuration-recorder name=default,roleARN=arn:aws:iam::123456789012:role/AWSConfigRole
+   ```
+
+4. **Start the Configuration Recorder:**
+   Finally, start the configuration recorder to begin recording configurations.
+   ```sh
+   aws configservice start-configuration-recorder --configuration-recorder-name default
+   ```
+
+By following these steps, you ensure that AWS Config is enabled and properly configured to monitor Route 53 and other AWS resources.
+</Accordion>
+
+<Accordion title='Using Python'>
+To prevent the misconfiguration of AWS Config not being enabled in Route 53 using Python scripts, you can follow these steps:
+
+### Step 1: Set Up AWS SDK for Python (Boto3)
+First, ensure you have Boto3 installed. If not, you can install it using pip:
+```bash
+pip install boto3
+```
+
+### Step 2: Create a Boto3 Session
+Create a session to interact with AWS services. This will allow you to use your AWS credentials and region settings.
+```python
+import boto3
+
+# Create a session using your AWS credentials
+session = boto3.Session(
+    aws_access_key_id='YOUR_ACCESS_KEY',
+    aws_secret_access_key='YOUR_SECRET_KEY',
```

Comment:
<b>Secret:</b> Secret Keyword <br/>
                    <b><a href="https://console.cloudanix.com/code?t=secrets" target='_blank'>Click here for more details</a></b>